docs/java/java-webapp.md

```diff
@@ -11,83 +11,74 @@ MetaDescription: Java web app tutorial showing how to deploy a Java web app to A
+- [Apache Maven](https://maven.apache.org), version 3.0 or above.
 
-<a class="tutorial-next-btn" href="https://azure.microsoft.com/pricing/free-trial/" target="_blank" style="background-color:#68217A">Create your free Azure account</a>
+>**Important**: The `JAVA_HOME` environment variable must be set to the install location of the JDK to complete this tutorial.
 
-The [Azure Command-Line Interface (CLI)](https://docs.microsoft.com/cli/azure/overview):
 
-<a class="tutorial-next-btn" href="https://docs.microsoft.com/cli/azure/install-azure-cli" target="_blank" style="background-color:#68217A">Install Azure CLI 2.0</a>
+## Install the Azure App Service extension
 
-## Login with Azure CLI
+The [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) extension is used to create, manage, and deploy to Azure App Service with key features including:
 
-To run the Azure CLI, we'll use the [Integrated Terminal](/docs/editor/integrated-terminal.md) in VS Code. To open the terminal, you can either:
+- Create new Azure Web App/deployment slot
+- Deploy to Azure Web App/deployment slot
+- Start, stop, and restart the Azure Web App/deployment slot
+- View a Web App's log files
+- Swap deployment slots
 
-* Use the `kb(workbench.action.terminal.toggleTerminal)` keyboard shortcut with the backtick character.
-* Use the **View** | **Integrated Terminal** menu command.
-* From the **Command Palette** (`kb(workbench.action.showCommands)`), use the **View: Toggle Integrated Terminal** command.
+To install the Azure App Service extension, open the Extensions view (`kb(workbench.view.extensions)`) and search for `azure app service` to filter the results. Select the Microsoft [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) extension. For a more command line Maven-centric experience, you can also check out the [Maven plugin for Azure Web App tutorial](https://docs.microsoft.com/en-us/java/api/overview/azure/maven/azure-webapp-maven-plugin/readme?view=azure-java-stable). 
 
-> **Note:** You can also open an external shell with the Explorer **Open in Command Prompt** command (**Open in Terminal** on macOS or Linux) if you prefer to work outside VS Code.
+## Create a new Web App 
 
-Sign into your Azure account by using the Azure CLI:
+Once the extension is installed, you can take the following steps to create a new Web App. 
 
-```bash
-az login
-```
+1. Click **Create New Project** button on the **APP SERVICE** Explorer view.
+2. Select a subscription. 
+3. Enter an unique name for the new Web App. 
+4. Select a location for the new Web App. 
+5. Select the OS as `Linux`. 
+6. Select the runtime of the Web App, for example`Tomcat 8.5`. 
 
-Follow the instructions to complete the sign-in process.
+> **Note:** The runtimes are in preview status for `Tomcat 8.5 (JRE8)`, `Tomcat 9.0 (JRE8)` and `Java SE (JRE8)`. 
```

Comment:
All the runtimes `Tomcat 8.5 (JRE8)`, `Tomcat 9.0 (JRE8)` and `Java SE (JRE8)` are in preview status.